src/index.js

```diff
@@ -26,6 +26,7 @@ export default ({
   return ({ dispatch, getState }) => next => action => {
     if (action[CALL_API]) {
       return dispatch({
+        isNativeCallApi: true,
```

Comment:
`revalidateDisabled: !action.isNativeCallApi` is a bit counterintuitive. How about changing the key to `revalidateDisabled` directly?

Reply:
Was thinking a way to separate if the action is the original `CALL_API` or `CHAIN_API` but I think it makes sense to make it consistent in this case. Thanks!